app/core/views.py

```diff
@@ -1,5 +1,6 @@
 from django.http import HttpResponse
 
 
+# pylint: disable=W0613
```

Comment:
Can also be:
`# pylint:disable=unused-argument`

Slightly clearer for the reader.